server/webapp/WEB-INF/rails.new/app/assets/javascripts/pipeline_configs/helpers/dragula_config.js

```diff
@@ -22,7 +22,10 @@ define(['dragula'], function (dragula) {
 
     var drake = dragula([elem], {
       revertOnSpill:   true,
-      mirrorContainer: elem
+      mirrorContainer: elem,
+      invalid: function(el, h) {
```

Comment:
Can we pass this as an option to `dragulaConfig`? Seems like the implementation is very specific to tasks